code/perception/src/opencv_lane_detection.py

```
@@ -0,0 +1,149 @@
+        self.publisher_image = self.new_publisher(
+            msg_type=numpy_msg(ImageMsg),
+            topic="/paf/hero/Lane_Detection_Image",
+            qos_profile=1,
+        )
+
+    def run(self):
+        self.spin()
+
+    def handle_camera_image(self, msg_image):
+        if self.device == "cuda":
+            if torch.cuda.is_available():
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)